src/transformers/models/levit/feature_extraction_levit.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2022 Meta Platforms, Inc. and The HuggingFace Inc. team. All rights reserved.
```

Comment:
The other file uses Meta Inc, here Meta Platforms, Inc is used.

I'd say let's use Meta Platforms, Inc. everywhere.

Reply:
Done